src/containers/login/index.js

```diff
@@ -1,2 +1,23 @@
 // @flow
-export { default } from "./login";
+import { connect } from "react-redux";
+import { withRouter } from "react-router-dom";
+import { login } from "../../actions/user";
+import Login from "./login";
+
+const mapStateToProps = ({ user }) => ({
+  error: user.error,
+  loading: user.loading,
+  auth: user.auth
+});
+
+const mapDispatchToProps = dispatch => ({
+  loginRequest: (password: string, loginName: string) =>
+    dispatch(login(password, loginName))
+});
```

Comment:
there is no need to re-map map dispatch to props. Just use the following:
`const mapDispatchToProps = {
  loginRequest: login,
};`